bitcoin/src/address/mod.rs

```diff
@@ -383,6 +383,9 @@ impl<V: NetworkValidation> Address<V> {
     pub fn as_unchecked(&self) -> &Address<NetworkUnchecked> {
         unsafe { &*(self as *const Address<V> as *const Address<NetworkUnchecked>) }
     }
+
+    /// Marks the network of this address as unchecked.
+    pub fn into_unchecked(self) -> Address { Address(self.0, PhantomData) }
```

Comment:
This function does not return an unchecked address, the generic defaults to `Address<NetworkChecked>`.

Reply:
```rust
    /// Converts the address back to an unchecked address.
    pub fn into_unchecked(self) -> Address<NetworkUnchecked> { Address(self.0, PhantomData) }
```

Reply:
Ugh, I can't believe I missed this. Thanks for the catch.